src/dotnet-setversion/Options.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using CommandLine;
+using CommandLine.Text;
+
+namespace dotnet_setversion
+{
+    public class Options
+    {
+        [Option('r', "recursive", SetName = "csprojFile", Default = false, HelpText =
```

Comment:
I haven't used CommandLineParser before. Is `SetName = "csprojFile"` correct here? What does it do?

Reply:
SetName allows to define a set of mutually exclusive options. However it's not applicable to parameters ([Value] in terms of CommandLineParser), so I couldn't make use of that feature in this case. I forget to remove it...